app/views/index/rss.phtml

```diff
@@ -3,16 +3,22 @@
 	/** @var FreshRSS_View $this */
 ?>
 <?= '<?xml version="1.0" encoding="UTF-8" ?>'; ?>
-<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:media="http://search.yahoo.com/mrss/"
+<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:media="http://search.yahoo.com/mrss/" xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd"
 	<?= $this->rss_base == '' ? '' : ' xml:base="' . $this->rss_base . '"' ?>
 >
 	<channel>
 		<title><?= $this->rss_title ?></title>
 		<link><?= $this->html_url ?></link>
-		<description><?= _t('index.feed.rss_of', $this->rss_title) ?></description>
+		<description><?= $this->description ?></description>
 		<pubDate><?= date('D, d M Y H:i:s O') ?></pubDate>
 		<lastBuildDate><?= gmdate('D, d M Y H:i:s') ?> GMT</lastBuildDate>
 		<atom:link href="<?= $this->rss_url ?>" rel="self" type="application/rss+xml" />
+		<itunes:image href="<?= $this->image_url ?>"></itunes:image>
```

Comment:
Is the `<itunes:image />` needed at all when there already is the `<image>` one?

Reply:
im honestly not sure. i added it because it was in the feed i was working off. my only testing podcast player is Overcast and it works without it but also the dev seems to have a very good understanding of the podcast RSS standards and makes sure Overcast can handle things so im not sure other apps will be as robust. there seem to be a whole range of "itunes" tags that are variously used in different feeds. there is also one for description `<itunes:summary>`.

i just tested with apple podcasts and it also works without the itunes tag, so i think it would be reasonable to remove it. i cant test any android podcast apps since i only have an iphone.

Reply:
Yes please for the test. I would rather add it again later if we identify a need

Reply:
cool, ill remove it and the itunes xml definition